pkgs/by-name/pc/pcsc-safenet/package.nix

```diff
@@ -4,24 +4,22 @@
 , autoPatchelfHook
 , dpkg
 , gtk3
-, openssl_1_1
+, openssl
 , pcsclite
 }:
 
 stdenv.mkDerivation rec {
   pname = "pcsc-safenet";
-  version = "10.8.28";
+  version = "10.8.1050";
 
-  debName = "Installation/Standard/Ubuntu-2004/safenetauthenticationclient_${version}_amd64.deb";
+  debName = "Installation/Standard/Ubuntu-2204/safenetauthenticationclient_${version}_amd64.deb";
 
   # extract debian package from larger zip file
-  src =
-    let
-      versionWithUnderscores = builtins.replaceStrings ["."] ["_"] version;
-    in fetchzip {
-      url = "https://www.digicert.com/StaticFiles/SAC_${versionWithUnderscores}_GA_Build.zip";
-      hash = "sha256-7XWj3T9/KnmgQ05urOJV6dqgkAS/A2G7efnqjQO2ing=";
-    };
+  src = fetchzip {
+    # URL version name is different that the version name of the .deb file inside
+    url = "https://www.digicert.com/StaticFiles/Linux_SAC_10_8_R1_GA.zip";
```

Comment:
Are you sure "R1_GA" will always be patch version 1050?

Reply:
Well, so far they have been putting each new version with a new file name, so I assume they will continue.

The latest version is just distributed as 10.8 R1 for some reason (most likely because Windows versions have been 10.8 R? for a while, up to R8 now). On the other hand, the .deb file in the archive, as well as the .so files in it, say 10.8.1050.

So the version essentially has two different names: R1 and 1050. I went with 1050 for the package name.